src/SNAP/pair_snap.cpp

```diff
@@ -233,66 +162,22 @@ void PairSNAP::compute_regular(int eflag, int vflag)
       }
     }
 
-    // compute Ui, Zi, and Bi for atom I
+    // compute Ui, Yi for atom I
 
     snaptr->compute_ui(ninside);
```

Comment:
@athomps Isn't this redundant if `quadratic_flag` is set? In other words it should be conditional on `!quadratic_flag`?